nilearn/maskers/tests/conftest.py

```diff
@@ -7,9 +7,7 @@
 
 def check_valid_for_all_maskers():
     """Return list of names of sklearn checks valid for all maskers."""
-    return [
-        "check_estimators_unfitted",
-    ]
+    return []
```

Comment:
do we still need this one if it's only returning empty list?

Reply:
no we don't 
thanks for catching this one